src/app/shared-component/status-drawer.component.ts

```diff
@@ -10,7 +10,7 @@ import { WorkItem } from './../work-item/work-item';
 @Component({
 	selector   : 'status-drawer',
 	templateUrl: './status-drawer.component.html',
-	styleUrls  : ['src/app/shared-component/status-drawer.component.css'],
+	styleUrls  : ['/status-drawer.component.css'],
```

Comment:
This seemed not to be working. Did you test this @baijum ? Also, why will not this path work on production? I believe it's a local reference to the app which is constant.